docs/base/rust_plugin.rst

```diff
@@ -0,0 +1,140 @@
+Rust plugin
+=============
+
+The Rust plugin can be used for Rust projects that use the Cargo build system.
+
+Keywords
+--------
+
+This plugin uses the common :ref:`plugin <part-properties-plugin>` keywords as
+well as those for :ref:`sources <part-properties-sources>`.
+
+Additionally, this plugin provides the plugin-specific keywords defined in the
+following sections.
+
+rust-channel
+~~~~~~~~~~~~
+**Type:** string
+**Default:** stable
+
+Used to select which Rust channel or version to use.
```

Comment:
It might be valuable to link to the [upstream documentation](https://rust-lang.github.io/rustup/concepts/channels.html) and let the user know they can specify either a channel or a specific toolchain.

Reply:
Done.